internal/app/app_test.go

```diff
@@ -46,10 +46,13 @@ func TestWait(t *testing.T) {
 		applyArgs    []string
 		noMethodCall bool
 
-		wantErr bool
+		wantAuthRejected bool
+		wantErr          bool
 	}{
 		"Cleanly exit on correct apply arguments": {applyArgs: []string{"http://proxy:3128", "", "", "", "", ""}},
 		"Timeout when no method is called on app": {noMethodCall: true},
+
```

Comment:
And so, here an additional test with multiple clients (both success, one success, one failure, and both failing)